app/javascript/components/FacetsPanel.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import FacetControl from './FacetControl';
+import MoreFacetsButton from './MoreFacetsButton';
+
+import { fetchFacets } from 'lib/scp-api';
+
+const defaultFacetIds = ['disease', 'organ', 'species', 'cell_type'];
+const moreFacetIds = ['sex', 'race', 'library_preparation_protocol', 'organism_age'];
+
+export default function FacetsPanel() {
+  const [defaultFacets, setDefaultFacets] = useState([]);
+  const [moreFacets, setMoreFacets] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const facets = await fetchFacets();
+      const df = facets.filter(facet => defaultFacetIds.includes(facet.id));
```

Comment:
this could be rewritten in lodash 
const df = _filter(facets, {id: defaultFacetIds})
(I don't feel too strongly about this change -- the lodash formulation is shorter, but sometimes just using plain JS is nice)

Reply:
Thanks for putting this Lodash convenience function on my radar.  I'll keep this one-liner in standard JavaScript since it's OK with you; I might try my hand with the Lodash version later.